internal/controller/velero.go

```diff
@@ -364,6 +364,10 @@ func (r *DataProtectionApplicationReconciler) customizeVeleroDeployment(veleroDe
 	disableInformerCache := disableInformerCacheValue(dpa)
 	veleroContainer.Args = append(veleroContainer.Args, fmt.Sprintf("--disable-informer-cache=%s", disableInformerCache))
 
+	if dpa.Spec.Configuration.Velero.ItemBlockWorkerCount > 0 {
```

Comment:
Should this check `dpa.Spec.Configuration.Velero.ItemBlockWorkerCount > 1` ?

Reply:
@shubham-pampattiwar I think `> 0` is what we want -- if the user wants to explicitly set it to 1, that's fine. Also, this doesn't require code changes if velero changes the default value in the future.